enarx-keep/build.rs

```diff
@@ -30,21 +33,56 @@ fn rerun_src(path: impl AsRef<Path>) -> std::io::Result<()> {
     Ok(())
 }
 
+fn create_dir(path: &PathBuf) {
+    match std::fs::create_dir(&path) {
+        Err(e) if e.kind() == std::io::ErrorKind::AlreadyExists => {}
+        Err(e) => {
+            eprintln!("Can't create {:#?} : {:#?}", path, e);
+            std::process::exit(1);
+        }
```

Comment:
@npmccallum I changed the code to use `std::fs::create_dir_all()` instead of this separate function based on @haraldh 's suggestion, since the former will not error if a directory already exists. Is there a reason we should *not* do it that way?

Reply:
Just kidding, I ended up leaving the original code as-is.